### PR TITLE
AJAX style tweaks

### PR DESCRIPTION
**Pull request by davidkpiano**
jun-08-2017 17-05-43

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -125,24 +121,38 @@
 
 .glimpse-ajax-rows {
     font-size: 13px !important;
-    padding-left: 20px !important;
-    margin-top: 1px !important;
+    margin-left: 20px !important;
+    width: 220px !important;
```
Comment (avanderhoorn) [inline src/index.scss]: What happens with this if the URL is too long? Does it still get trimmed?

File: src/views/ajax.js [hunk 2]
```diff
@@ -38,14 +38,14 @@ function update(details) {
         dom.removeClass(counter, 'glimpse-section-value--update');
     }, 2000);
 
-    //manage row values
-    if (state.summaryStack.length === 0) {
-        var section = document.getElementById('glimpse-ajax-summary');
-        section.insertAdjacentHTML('beforeend', '<div class="glimpse-ajax-rows" id="glimpse-ajax-rows"></div>');
-    }
-    recordItem(rowTemplate(details), document.getElementById('glimpse-ajax-rows'), state.summaryStack, 2);
+    state.summaryStack.push(details);
```
Comment (avanderhoorn) [inline src/views/ajax.js]: It looks like we are going to keep pushing items into this array for ever. What `recordItem` used to do was make sure that the array only had what we needed. Also the idea behind `recordItem` was that the same logic can be reused for the popup window.

File: src/views/ajax.js [hunk 3]
```diff
@@ -38,14 +38,14 @@ function update(details) {
...
+
+    document.getElementById('glimpse-ajax-rows').innerHTML = state.summaryStack
+        .slice(-2)
+        .map(rowTemplate)
+        .join('\n');
 }
-var recordItem = function(html, container, stack, length) {
+function recordItem(html, container, stack, length) {
```
Comment (avanderhoorn) [inline src/views/ajax.js]: If it turns out that we aren't using this any more we can probably remove it.

**Comment by avanderhoorn** [comment]
Also can we remove the `(` and `)` around the number and make the number big?

File: src/index.scss (hunk 1 above)
Comment (davidkpiano) [reply src/index.scss]: Yes, it does (with ellipses on right), e.g. `foo/bar/...`

Comment (avanderhoorn) [reply src/index.scss]: Cool!

**Comment by avanderhoorn** [comment]
LGTM

**Pull request merged**